lib/routes/token.js

```diff
@@ -160,7 +160,8 @@ module.exports = {
       scope: validators.scope.required().allow(''),
       token_type: Joi.string().valid('bearer').required(),
       expires_in: Joi.number().max(MAX_TTL_S).required(),
-      auth_at: Joi.number()
+      auth_at: Joi.number(),
+      derivedKeyBundle: Joi.string().optional()
```

Comment:
In [1] the name of this field is "keys_jwe", by symmetry with the "keys_jwk" parameter passed in at the start of the dance.

[1] https://docs.google.com/document/d/1IvQJFEBFz0PnL4uVlIvt8fBS_IPwSK-avK0BRIHucxQ/edit#